model_compression_toolkit/core/pytorch/back2framework/quantization_wrapper/quantized_layer_wrapper.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import copy
```

Comment:
@ofirgo, please review this code and see if it can be use for MP in Pytorch
@eladc-git, please review this code and see if it can be use for GPTQ in Pytorch